src/wasm-lib/kcl/src/std/patterns.rs

```diff
@@ -1024,21 +1099,38 @@ pub async fn pattern_circular_3d(exec_state: &mut ExecState, args: Args) -> Resu
 ///   |> circle({ center = [0, 0], radius = 1 }, %)
 ///
 /// example = extrude(exampleSketch, length = -5)
-///   |> patternCircular3d({
+///   |> patternCircular3d(
 ///        axis = [1, -1, 0],
 ///        center = [10, -20, 0],
 ///        instances = 11,
 ///        arcDegrees = 360,
 ///        rotateDuplicates = true
-///      }, %)
+///      )
 /// ```
 #[stdlib {
     name = "patternCircular3d",
     feature_tree_operation = true,
+    keywords = true,
+    unlabeled_first = true,
+    args = {
+        solid_set = { docs = "Which solid(s) to pattern" },
+        instances = { docs = "The number of total instances. Must be greater than or equal to 1. This includes the original entity. For example, if instances is 2, there will be two copies -- the original, and one new copy. If instances is 1, this has no effect."},
+        axis = { docs = "The axis around which to make the pattern. This is a 3D vector"},
+        center = { docs = "The center about which to make the pattern. This is a 3D vector."},
+        arc_degrees = { docs = "The arc angle (in degrees) to place the repetitions. Must be greater than 0."},
+        rotate_duplicates= { docs = "Whether or not to rotate the duplicates as they are copied."},
+        use_original= { docs = "If the target was sketched on an extrusion, setting this will use the original sketch as the target, not the entire joined solid. Defaults to false."},
```

Comment:
```suggestion
        rotate_duplicates = { docs = "Whether or not to rotate the duplicates as they are copied."},
        use_original = { docs = "If the target was sketched on an extrusion, setting this will use the original sketch as the target, not the entire joined solid. Defaults to false."},
```